### PR TITLE
feat: add radio and checkbox

### PR DESCRIPTION
**Pull request by jefrydco**
## Description

https://github.com/Jitera/jitera-frontend/pull/1676

### CODE REVIEW

File: src/components/atoms/Checkbox/Checkbox.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+
+import { CheckboxStateColor } from '@src/types/component'
+
+export const CheckboxWrapper = styled.div<CheckboxStateColor>`
```
Comment (quy-truong) [inline src/components/atoms/Checkbox/Checkbox.styles.tsx:5]: I thought we don't use styled-component for exported project? 🤔

File: src/components/atoms/Checkbox/Checkbox.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    .jitera-checkbox-wrapper:hover .jitera-checkbox,
+    .jitera-checkbox:hover .jitera-checkbox-inner,
+    .jitera-checkbox-input:focus + .jitera-checkbox-inner {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
+    }
+    .jitera-checkbox-checked::after {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
+    }
+    .jitera-checkbox-checked .jitera-checkbox-inner {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
```
Comment (quy-truong) [inline src/components/atoms/Checkbox/Checkbox.styles.tsx:19]: should we move `'#1890ff'` to `constant.ts`

Comment (jefrydco) [reply src/components/atoms/Checkbox/Checkbox.styles.tsx:19]: Yup, I'll move it

File: src/components/atoms/Checkbox/Checkbox.styles.tsx (hunk 1 above)
Comment (jefrydco) [reply src/components/atoms/Checkbox/Checkbox.styles.tsx:5]: Ah right, initially, I saw @duongjitera use styled component for Visibility thing, I just imitated it 🤔

**Review by quy-truong — APPROVED** [review]

**Pull request merged**